src/cpu/x64/brgemm/report_op.xml

```diff
@@ -0,0 +1,250 @@
+		<Relu-1 opsets="1 10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<ReorgYolo-2 opsets="10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<Reshape-1 opsets="1 10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<Result-1 opsets="1 10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<Reverse-1 opsets="1" />
+		<ReverseSequence-1 opsets="1 10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<Roll-7 opsets="10 11 12 13 14 15 7 8 9" />
+		<Round-5 opsets="10 11 12 13 14 15 5 6 7 8 9" />
+		<STFT-15 opsets="15" />
+		<ScaledDotProductAttention-13 opsets="13 14 15" />
+		<ScatterElementsUpdate-12 opsets="12 13 14 15" />
+		<ScatterElementsUpdate-3 opsets="10 11 3 4 5 6 7 8 9" />
+		<ScatterNDUpdate-15 opsets="15" />
+		<ScatterNDUpdate-4 opsets="10 11 12 13 14 4 5 6 7 8 9" />
+		<ScatterUpdate-3 opsets="10 11 12 13 14 15 3 4 5 6 7 8 9" />
+		<SearchSorted-15 opsets="15" />
+		<Select-1 opsets="1 10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<Selu-1 opsets="1 10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<ShapeOf-1 opsets="1 2" />
+		<ShapeOf-3 opsets="10 11 12 13 14 15 16 3 4 5 6 7 8 9" />
+		<ShuffleChannels-1 opsets="1 10 11 12 13 14 15 3 4 5 6 7 8 9" />
+		<Sigmoid-1 opsets="1 10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<Sign-1 opsets="1 10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<Sin-1 opsets="1 10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<Sinh-1 opsets="1 10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<Slice-8 opsets="10 11 12 13 14 15 8 9" />
+		<SliceScatter-15 opsets="15" />
+		<SoftPlus-4 opsets="10 11 12 13 14 15 4 5 6 7 8 9" />
+		<SoftSign-9 opsets="10 11 12 13 14 15 9" />
+		<Softmax-1 opsets="1 2 3 4 5 6 7" />
+		<Softmax-8 opsets="10 11 12 13 14 15 8 9" />
+		<SpaceToBatch-2 opsets="10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<SpaceToDepth-1 opsets="1 10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<Split-1 opsets="1 10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<Sqrt-1 opsets="1 10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<SquaredDifference-1 opsets="1 10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<Squeeze-1 opsets="1 10 11 12 13 14 2 3 4 5 6 7 8 9" />
+		<Squeeze-15 opsets="15" />
+		<StridedSlice-1 opsets="1 10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<StringTensorPack-15 opsets="15" />
+		<StringTensorUnpack-15 opsets="15" />
+		<Subtract-1 opsets="1 10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<Swish-4 opsets="10 11 12 13 14 15 4 5 6 7 8 9" />
+		<Tan-1 opsets="1 10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<Tanh-1 opsets="1 10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<TensorIterator-1 opsets="1 10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<Tile-1 opsets="1 10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<TopK-1 opsets="1 2" />
+		<TopK-11 opsets="11 12 13 14 15" />
+		<TopK-3 opsets="10 3 4 5 6 7 8 9" />
+		<Transpose-1 opsets="1 10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<Unique-10 opsets="10 11 12 13 14 15" />
+		<Unsqueeze-1 opsets="1 10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<VariadicSplit-1 opsets="1 10 11 12 13 14 15 2 3 4 5 6 7 8 9" />
+		<Xor-1 opsets="1" />
+	</ops_list>
+</report>
```

Comment:
Is this file neccessary？

Reply:
Definetelly not. Thanks for pointing that out. Removed